src/Angular.js

```diff
@@ -232,8 +232,7 @@ function isArrayLike(obj) {
 
   // NodeList objects (with `item` method) and
   // other objects with suitable length characteristics are array-like
-  return isNumber(length) &&
-    (length >= 0 && ((length - 1) in obj || obj instanceof Array) || typeof obj.item === 'function');
+  return isNumber(length) && (length >= 0 && (length - 1) in obj || typeof obj.item === 'function');
```

Comment:
`isArray` is used in a couple of lines above. So if `obj instanceof Array`, the function hits return on that line.